### PR TITLE
fix RBAC in capabilities package

### LINKED ISSUE #3688: Capabilities package RBAC rules are not updated
**Bug description**
After capabilities controller was updated to reconcile on Capabilities resources in both run.tanzu.vmware.com and core.tanzu.vmware.com, we missed updating the RBAC rules in the Capabilities package.

**Affected product area (please put an X in all that apply)**

- ( ) APIs
- (x) Addons
- ( ) CLI
- ( ) Docs
- ( ) IAM
- ( ) Installation
- ( ) Plugin
- ( ) Security
- ( ) Test and Release
- ( ) User Experience
- ( ) Developer Experience

**Expected behavior**

**Steps to reproduce the bug**

**Version** (include the SHA if the version is not obvious)

**Environment where the bug was observed (cloud, OS, etc)**

**Relevant Debug Output (Logs, manifests, etc)**

### PR DESCRIPTION
**Pull request by yharish991**
Signed-off-by: Harish Yayi <yharish991@gmail.com>

### What this PR does / why we need it
This PR fixes the RBAC rules in capabilities package

### Which issue(s) this PR fixes

Fixes https://github.com/vmware-tanzu/tanzu-framework/issues/3688

### Describe testing done for PR

Generated capabilities package and tested it by installing it in the cluster
### Release note

```release-note
NONE
```

### CODE REVIEW

File: featuregates/client/go.mod [hunk 1]
```diff
@@ -11,6 +11,7 @@ replace (
 	github.com/vmware-tanzu/tanzu-framework/cli/core => ../../cli/core
 	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ../../cli/runtime
 	github.com/vmware-tanzu/tanzu-framework/packageclients => ../../packageclients
+	github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkr => ../../pkg/v1/tkr
```
Comment (sathyanarays) [inline featuregates/client/go.mod:14]: Is this change necessary?

Comment (yharish991) [reply featuregates/client/go.mod:14]: this is an unrelated change, but the featuregates client module was complaining about this.. since it is a small change, I thought I will push it with this change.

**Review by codegold79 — APPROVED** [review]

**Review by danniel1205 — APPROVED** [review]
lgtm, thanks for the PR
